lib/optimizely.rb

```diff
@@ -1095,11 +1097,11 @@ def send_impression(config, experiment, variation_key, flag_key, rule_key, enabl
       @event_processor.process(user_event)
       return unless @notification_center.notification_count(NotificationCenter::NOTIFICATION_TYPES[:ACTIVATE]).positive?
 
-      @logger.log(Logger::INFO, "Activating user '#{user_id}' in experiment '#{experiment_key}'.")
+      @logger.log(Logger::INFO, "Activating user '#{user_id}' in experiment '#{experiment_id}'.")
```

Comment:
keep experiment_key in log

Reply:
I agree with @jaeopt on this. Also adding experiment_id can actually help too

Reply:
Having both will be helpful, but for consistency with all other SDKs, let's stick to "key" only in logs and messages